SPDY/SPDYSocket.m

```diff
@@ -605,7 +635,31 @@ - (bool)_connectToNextEndpointWithError:(NSError **)pError
     return YES;
 
     Failed:
-    return NO;
+    if (pError) {
+        *pError = nil;
+    }
+    return [self _connectToNextEndpointWithError:pError];
+}
+
+- (BOOL)_attemptNextProxyConnectionForError:(NSError *)rootError
```

Comment:
Looks like we don't use the returned value of this method anywhere. Also, given the places it's called I'd suggest a more generic name, e.g.:

- (void)_handleError:(NSError *)error